hw02-generics-collections/src/main/java/ru/otus/kovaleva/CustomerReverseOrder.java

```diff
@@ -0,0 +1,17 @@
+package ru.otus.kovaleva;
+
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+public class CustomerReverseOrder {
+
+    private Deque<Customer> customers = new ArrayDeque<>();
+
+    public void add(Customer customer) {
+        customers.push(customer);
+    }
+
+    public Customer take() {
+        return customers.isEmpty() ? null : customers.pop();
```

Comment:
Есть метод `poll()` - можно его использовать:
* `pop()` бросает исключение, если нет элементов
* `poll()` возвращает `null`, если нет элементов

```suggestion
        return customers.poll();
```